chayce-solchaga.txt

```diff
@@ -0,0 +1,114 @@
+        in the <a> tag, you add an attribute "target="_blank""
+    14. What is the alt attribute in the image tag used for? 
+        The alt attrib is used to describe the image to give context to screen readers, users with accessibility issues, and provides alt-text if the image is unable to load for some reason. 
+    15. How do I reference an id?
+        Using a hashtag #
+    16. What is the difference between a section and a div
+        A section is more for organizing content semantically while a div is usueful for organizing the layout and presentation of such content. 
+    17. What is CSS used for?
+        CSS is used to style your HTML document 
+    18. How to we select an element? Example - every h2 on the page
+        h2 {}
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one
+        The main difference is a class allows you to group and style a group of similar elements while an ID us used more for a specific element that is unique since best practices say you should only use it once. An example of this is if I have 4 div's on my page. All the div's need orange text, but on one day, I need to change the test red on only one of the boxes. This is when I would use an ID to allow me to make changes to 1 box universally without having to edit any other code. 
+    20. How do we select classes in CSS?
+        the . - .class
+    21. How do we select a p element with a single class of “human””?
+        .human, or .human p 
+    22. What is a parent child selector? When would this be useful? 
+        A parent child selector is a way to grab a specific type of element within something such as a class to style that element specifically. For example, you could have p's all over the page, but in one specific section with a class of ".books", you need to change the font, you would be able to select this specific p tag by using the parent child selector of ".books p."
+    23. How do you select all links within a div with the class of sidebar?
+        .sidebar a
+    24. What is a pseudo selector?
+        A pseudo selector allows you to style an element based on a specific state. A great example on this is in links. When we style a link, we use a{}. However, if we wanted to change the color when the link is hovered by the mouse, we would use a:hover{}.
+    25. What do we use the change the spacing between lines?
+        line-height property
+    26. What do we use to change the spacing between letters?
+        letter-spacing
+    27. What do we use to to change everything to CAPITALS? lowercase? Capitalize?
+        text-transform: uppercase, lowercase, capitalize;
+    28. How do I add a 1px border around my div that is dotted and black?
+        border-width: 1px;
+        border-style: dotted;
+        border-color: black;
+    29. How do I select everything on the page? 
+        CMD+A on mac (or if this is a CSS page you can use html{})
```

Comment:
I think what they were looking for was the Wildcard selector here. I like the Control A answer tho 👍